### PR TITLE
[CDAP-12526] Add new configurations introduced in Tephra 0.13.0

### PR DESCRIPTION
**Pull request by anew**
New configurations from Tephra 13:
- limits and warn thresholds for the size (number of changes and size in bytes) of a change set
- how long to retain the client id of a tx (to include it in log messages and conflict exceptions)

### CODE REVIEW

**Review by poornachandra — APPROVED** [review]

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -777,19 +822,23 @@
...
 
   <property>
-    <name>data.tx.pruning.plugin.class</name>
-    <value>co.cask.data2.txprune.DefaultHBaseTransactionPruningPlugin</value>
+    <name>data.tx.retain.client.id</name>
+    <value>committed</value>
     <description>
-      Class name for the default transaction pruning plugin
+      Whether and how long to retain the client id of a transaction. Valid values are:
+      "off" to disable retention of client ida; "active" to retain the client id until
```
Comment (anwar6953) [inline cdap-common/src/main/resources/cdap-default.xml]: s/ida/ids

Remainder LGTM

**Comment by anew** [comment]
thanks for the review, fixed the typo, will squash and merge.

**Comment by anew** [comment]
Actually I had to fix a failing test case. @poornachandra please review the last commit.

**Comment by anew** [comment]
New build running here: https://builds.cask.co/browse/CDAP-RUT1379-5

**Review by poornachandra — APPROVED** [review]

**Pull request merged**